pkg/testdata/models/apis/iam/0000-00-00/generator.yaml

```diff
@@ -0,0 +1,37 @@
+ignore:
+  resource_names:
+   - AccessKey
+   - AccountAlias
+   - Group
+   - InstanceProfile
+   - LoginProfile
+   - OpenIDConnectProvider
+   - Policy
+   - PolicyVersion
+   #- Role
+   - SAMLProvider
+   - ServiceLinkedRole
+   - ServiceSpecificCredential
+   - User
+   - VirtualMFADevice
+resources:
+  Role:
+    renames:
+      operations:
+        CreateRole:
+          input_fields:
+            RoleName: Name
+        GetRole:
+          input_fields:
+            RoleName: Name
+        UpdateRole:
+          input_fields:
+            RoleName: Name
+        DeleteRole:
+          input_fields:
+            RoleName: Name
+    fields:
+      PermissionsBoundary:
+        set:
```

Comment:
Does the interface require an operation to be set? It looks to be the case [here](https://github.com/aws-controllers-k8s/code-generator/blob/main/pkg/generate/ack/controller.go#L112)

Reply:
If `method` is missing, it means all operations :)

https://github.com/aws-controllers-k8s/code-generator/blob/d56efd462bcb087428d527e6a6a220caa6298eba/pkg/generate/config/field.go#L152-L157